samples/filter-features-in-scene/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application><activity
+            android:exported="true"
+            android:name=".MainActivity"
+            android:label="@string/display_composable_map_view_app_name">
```

Comment:
I think there may a bug with the new module script here, but `label` string needs a change